packages/synapse-react-client/src/components/EntityUpload/EntityUpload.tsx

```diff
@@ -201,7 +201,6 @@ export const EntityUpload = forwardRef(function EntityUpload(
             />
             {/* File input */}
             <input
-              className="form-control"
               type="file"
               id="filesToUpload"
               name="file upload"
```

Comment:
These `input`s with `form-control` are not visible, so removing the class doesn't matter